### DISCUSSION TITLE
Log localtime

### DISCUSSION THREAD

**Discussion by agabellini** [ant-media/Ant-Media-Server#6608] [Q&A]
Hello,

I would like to have the logs in localtime and not UTC. Is it possible? How can I do it?

Thanks,
Andrea

**Comment by SelimEmre** [comment]
Hi @agabellini,

I'm a bit confused. Ant Media Server already logs using the instance's local time. However, you can change the time format in `/usr/local/antmedia/conf/logback.xml`.

The default setting is:
```
<pattern>%d{ISO8601} [%thread] %-5level %logger{35} - %msg%n</pattern>
```

You can change the setting to:
```
<pattern>%d{yyyy-MM-dd HH:mm:ss.SSS Z} [%thread] %-5level %logger{35} - %msg%n</pattern>
```

**Comment by agabellini** [comment]
It's strange for me too... I installed AMS on Ubuntu 24.04, and both globally and for the antemedia user the settings are these:

```
[09:50:08][antmedia@streaming02:~]$date
Mon Aug 26 09:50:09 CEST 2024
[09:50:09][antmedia@streaming02:~]$localectl
System Locale: LANG=en_US.UTF-8
               LC_TIME=C.UTF-8
    VC Keymap: (unset)
   X11 Layout: it
    X11 Model: pc105
```

but the logs continue to be in UTC

Would it be possible to send the logs via syslog?

**Comment by agabellini** [comment]
Hello,

I solved with an override to the systemd unit file of the service antmedia:

```
[Service]
Environment=TZ=Europe/Rome
```
